google_symptoms/delphi_google_symptoms/run.py

```diff
@@ -46,7 +47,14 @@ def run_module(params):
     export_start_date = datetime.strptime(
         params["indicator"]["export_start_date"], "%Y-%m-%d")
     export_dir = params["common"]["export_dir"]
-    num_export_days = params["indicator"].get("num_export_days", "all")
+
+    if "num_export_days" in params["indicator"]:
```

Comment:
Why we have such a check here?

Reply:
Isn't the `num_export_days` deleted from the `params` file? If you want to check this, I think it is better to keep the `num_export_days` term in `params` but set it to `None` in default. Otherwise, this check does not make sense to me.